browser/resources/settings/brave_default_extensions_page/brave_default_extensions_page.html

```diff
@@ -174,6 +186,10 @@
         on-click="openWebStoreUrl_" external>
       </cr-link-row>
     </div>
+  <template is="dom-if" if="[[ showChangeIPFSGatewayDialog_]]" restamp>
```

Comment:
nit: formatting - extra space after `[[`

Reply:
Fixed in 6614869913a0157aba7701a13ab1ae73c4d3e6e6